docs/azure-marketplace.asciidoc

```diff
@@ -95,6 +95,22 @@ group. This is particularly useful in situations where resources may already be
 deployed in another resource group, and the cluster should be accessible to those
 resources on the same network, and possibly same subnet.
 
+[IMPORTANT]
```

Comment:
Wow had no clue! Is there a link to the official documentation that we can link here?

Reply:
Sadly not :(